src/ruby/integration/deploy_an_app_with_custom_gemfile_bad_dummy_gemfile_test.go

```diff
@@ -1,29 +0,0 @@
-package integration_test
-
-import (
-	"github.com/cloudfoundry/libbuildpack/cutlass"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe("App with custom Gemfile and bad file named 'Gemfile'", func() {
-	var app *cutlass.App
-
-	AfterEach(func() {
-		if app != nil {
-			app.Destroy()
-		}
-		app = nil
-	})
-
-	BeforeEach(func() {
-		app = cutlass.New(Fixtures("custom_gemfile_bad_dummy_gemfile"))
-	})
-
-	It("uses the version of ruby specified in Gemfile-APP", func() {
```

Comment:
fine to delete this test, but it feels like something we should be unit testing

Reply:
It is unit tested: https://github.com/cloudfoundry/ruby-buildpack/blob/40bdf71ca85ff0865012e464d15cc4f287d0e392/src/ruby/versions/ruby_test.go#L234-L255